### ISSUE TITLE
Python 3.12 build errors due to SafeConfigParser removal

### ISSUE THREAD

**Issue by vpa1977** [jdidion/atropos#135]
Python 3.12 removed SafeConfigParser causing the following build errors
```
I: pybuild base:310: python3.12 setup.py clean 
/<<PKGBUILDDIR>>/versioneer.py:485: SyntaxWarning: invalid escape sequence '\s'
  LONG_VERSION_PY['git'] = '''
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/setup.py", line 136, in <module>
    version=versioneer.get_version(),
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/<<PKGBUILDDIR>>/versioneer.py", line 1473, in get_version
    return get_versions()["version"]
           ^^^^^^^^^^^^^^
  File "/<<PKGBUILDDIR>>/versioneer.py", line 1406, in get_versions
    cfg = get_config_from_root(root)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/<<PKGBUILDDIR>>/versioneer.py", line 412, in get_config_from_root
    parser = configparser.SafeConfigParser()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?
E: pybuild pybuild:395: clean: plugin distutils failed with: exit code=1: python3.12 setup.py clean 
dh_auto_clean: error: pybuild --clean -i python{version} -p "3.12 3.11" returned exit code 13
make[1]: *** [debian/rules:24: override_dh_auto_clean] Error 25
```

See https://launchpadlibrarian.net/699337441/buildlog_ubuntu-noble-amd64.atropos_1.1.31+dfsg-3build3_BUILDING.txt.gz

**Comment by jdidion** [comment]
Thanks, it looks like `SafeConfigParser` was renamed to `ConfigParser` in python3 and I just never updated. Will make a new release with this fix.

**Comment by jdidion** [comment]
Fixed in 1.1.32
